code/src/layouts/header/components/header-search-input-dropdown/HeaderSearchInputDropdown.scss

```diff
@@ -0,0 +1,57 @@
+@import "@design-system";
+
+.search-input-dropdown {
+  padding: 0;
+  width: 270px;
+  position: absolute;
+  top: 30px;
+  left: 5px;
+  background-color: var(--spa-dropdown-background-color, $white);
+  border-radius: spacing(1);
+  box-shadow: $shadow-black-alpha;
+  z-index: $layer-dropdown;
+  height: 250px;
+  overflow-y: auto;
+
+  &__no-results {
+    height: 100%;
+    list-style: none;
```

Comment:
Use mixin for it, it is called `list-reset` I believe

Reply:
used